src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
+/* eslint-disable no-console */
 'use strict';
 
-// write code here
+function parseSalary(salary) {
+  if (!salary) {
+    return 0;
+  }
+
+  return +salary.replaceAll(/[$,.]/g, '') || 0;
```

Comment:
Move regexpt to constant ( you don't remember what meen this regexp after 1 month)